kamailio/registrar-role.cfg

```diff
@@ -281,11 +281,12 @@ route[SAVE_LOCATION]
     }
 
     $var(ip) = $Ri;
+    $var(port) = $Rp;
     if(af==INET6) {
        $var(ip) = "[" + $Ri + "]";
     }
 
-    $var(amqp_payload_request) = '{"Event-Category" : "directory", "Event-Name" : "reg_success", "Status" : "$var(Status)", "Event-Timestamp" : $TS, "Expires" : $(var(expires){s.int}), "First-Registration" : $var(new_reg), "Contact" : "$(ct{s.escape.common})", "Call-ID" : "$ci", "Realm" : "$fd", "Username" : "$fU", "From-User" : "$fU", "From-Host" : "$fd", "To-User" : "$tU", "To-Host" : "$td", "User-Agent" : "$(ua{s.escape.common})" , "Custom-Channel-Vars" : $xavp(ulattrs=>custom_channel_vars), "Proxy-Path" : "sip:$var(ip)", "RUID" : "$xavp(ulrcd=>ruid)" }';
+    $var(amqp_payload_request) = '{"Event-Category" : "directory", "Event-Name" : "reg_success", "Status" : "$var(Status)", "Event-Timestamp" : $TS, "Expires" : $(var(expires){s.int}), "First-Registration" : $var(new_reg), "Contact" : "$(ct{s.escape.common})", "Call-ID" : "$ci", "Realm" : "$fd", "Username" : "$fU", "From-User" : "$fU", "From-Host" : "$fd", "To-User" : "$tU", "To-Host" : "$td", "User-Agent" : "$(ua{s.escape.common})" , "Custom-Channel-Vars" : $xavp(ulattrs=>custom_channel_vars), "Proxy-Path" : "sip:$var(ip):$var(port)", "RUID" : "$xavp(ulrcd=>ruid)" }';
```

Comment:
kamailio is publishing all the info we need to know about and more :)
check latest stable version https://github.com/2600hz/kazoo-configs/blob/4.0.11/kamailio/registrar-role.cfg#L283

you may be looking for  `"Source-IP": "$si", "Source-Port": "$sp"` 

if you're looking for Proxy ip/port then you'll need to wait for 4.1 (only in master right now)
`"Proxy-IP" : "$var(ip)", "Proxy-Port" : "$Rp"`